src/verse.cpp

```diff
@@ -175,7 +176,11 @@ int Verse::AdjustSylSpacing(FunctorParams *functorParams)
 
     // Use the syl because the content bounding box of the verse might be invalid at this stage
     int overlap = params->m_lastSyl->GetContentRight() - (firstSyl->GetContentLeft() + xShift);
-    overlap += lastSyl->CalcConnectorSpacing(params->m_doc, params->m_staffSize);
+    overlap += params->m_lastSyl->CalcConnectorSpacing(params->m_doc, params->m_staffSize);
+
+    if (params->m_lastSyl->GetUuid() == "syl-0000001599481773") {
+        int d = params->m_doc->GetDrawingUnit(params->m_staffSize);
+    }
```

Comment:
??

Reply:
Sorry, this is about my further work. I'll remove it.